client-runtime/src/main/java/com/microsoft/rest/v2/SwaggerMethodParser.java

```diff
@@ -43,6 +46,7 @@
  * method.
  */
 public class SwaggerMethodParser {
+    private final SerializerAdapter<?> serializer;
```

Comment:
Is this for handling template substitutions on non-String parameters?

Reply:
Correct. Specifically non-String header values and non-String query values.